app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,2 +1,2 @@
 // Import page-specific CSS files here.
-@import "home";
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
It seems that you've already imported the font in _fonts.scss - can you double check?

Reply:
you're right, I updated the code.